docs/src/pages/components/tabs/UnstyledTabsBasic.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import Tabs from '@mui/base/TabsUnstyled';
+import TabsList from '@mui/base/TabsListUnstyled';
```

Comment:
The names don't match, I personally find it confusing 
```suggestion
import TabsListUnstyled from '@mui/base/TabsListUnstyled';
```

x the other if agreed on (it's base on the convention of the other demos)

Reply:
+1

Reply:
Done